test/models/review_test.rb

```diff
@@ -0,0 +1,55 @@
+require "test_helper"
+
+describe Review do
+  let(:review) { reviews(:goodone, :badone).sample }
+
+  describe "relationships" do
+    it "returns a product" do
+      review.product.must_be_instance_of Product
+      review.product.reviews.must_include review
+    end
+
+    it "product still exists if review deleted" do
+      review = Review.create(rating: 5, comment: "most perfect", product: products(:kidjams))
+      review.destroy
+      products(:kidjams).wont_be_nil
+    end
+  end
+
+  describe "validations" do
+    it "is valid with an integer rating 1-5" do
+      review.valid?
```

Comment:
When you want it to be valid - you still want to add the `must_equal` assertion, otherwise it won't tell you if it is invalid